hack/e2e/end_e2e_core.sh

```diff
@@ -12,7 +13,7 @@ while [ $n -le 60 ];do
     if_exist=$(docker exec k3d-k3s-default-server-0 sh -c "test -f $file && echo 'ok'")
     echo "$if_exist"
     if [ -n  "$if_exist" ];then
-        docker exec k3d-k3s-default-server-0 cat "$file" > /tmp/e2e-profile.out
+        docker exec k3d-k3s-default-server-0 cat "$file" >> /tmp/e2e-profile.out
```

Comment:
Why we need to change it here? \cc @chivalryq

Reply:
Because in the multicluster-e2e-test and apiserver-e2e-test, the sharding option is enabled and we have two controller logs to collect.